src/domain/events/eventList/EventList.tsx

```diff
@@ -161,8 +161,8 @@ const EventListContainer: React.FC<EventListContainerProps> = (props) => {
   const sortOptions = useEventSortOptions();
 
   const variables = {
-    ...baseVariables,
     ...getEventsQueryVariables(location.search),
+    ...baseVariables,
```

Comment:
Not sure if this is the desired effect after all, baseVariables would now overwrite whatever getEventsQueryVariables returns from the location(?) 😅 . Perhaps instead we should work the start "now" parameter into the location instead? 🤔